utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -505,7 +516,7 @@ pub struct SharedParams {
 		ignore_case = true,
 		default_value_t = DEFAULT_WASM_EXECUTION_METHOD,
 	)]
-	pub wasm_method: WasmExecutionMethod,
+	wasm_method: WasmExecutionMethod,
```

Comment:
I made these pub so they would show up in the rust docs.

Reply:
I failed to spot it when merging `master`. Reverted.